packit_service/service/events.py

```diff
@@ -710,8 +710,9 @@ def __init__(
     @property
     def tag_name(self):
         if not self._tag_name:
-            releases = self.project.get_releases()
-            self._tag_name = releases[0].tag_name if releases else ""
+            self._tag_name = ""
+            if latest_release := self.project.get_latest_release():
```

Comment:
I hope we don't break anything with this..;)

Reply:
base is F33, so we should be fine :)